src/fsharp/fcs-fable/adapters.fs

```diff
@@ -0,0 +1,269 @@
+namespace Internal.Utilities
```

Comment:
Please put this under ``src/frameworks/fable/adapters.fs``.  We can then put other framework adapters under ``src/frameworks/netcore/...`` etc. Thanks